DTS-STN/secure-client-hub
# got rid of duplicate modals

## [ADO-XXX](https://dev.azure.com/VP-BD/DECD/_workitems/edit/XXX)

### Description

List of proposed changes:

- Altered  SessionTimeout to use embedded modal component instead of creating a new modal instance. 
-

### What to test for/How to test
 - Click on link that would normally trigger beta modal. 
 - Wait 30 seconds for timer modal to appear. 


### Additional Notes
